doc/classes/OS.xml

```diff
@@ -538,6 +538,16 @@
 				Shows all resources currently used by the game.
 			</description>
 		</method>
+		<method name="register_protocol">
+			<return type="int" enum="Error" />
+			<argument index="0" name="protocol" type="String" />
+			<description>
+				Associates links that look like [code]protocol://uri[/code] with this executable.
+				When the OS opens a link like this, such as when the user clicks on a button in their web browser, the executable will launch with the given [code]uri[/code] as the [code]--uri[/code] command line argument, such as [code]myapp://launch/12345[/code] running [code]myapp.exe --uri="myapp://launch/12345"[/code].
+				Returns one of the [enum Error] code constants ([code]OK[/code] on success, [code]ERR_INVALID_PARAMETER[/code] if the given protocol is invalid).
+				[b]Note:[/b] This method is implemented on Linux and Windows, but on macOS you need to use the export dialog to register a protocol.
```

Comment:
I thought the goal was to move this entirely to ProjectSettings? See the discussion in https://github.com/godotengine/godot/pull/62788

Reply:
yep this is being refactored out I just haven't removed it

Reply:
All the code will be in the module but wanted to do this on monday